move2kube/application.properties

```diff
@@ -4,8 +4,9 @@ mp.messaging.incoming.kogito_incoming_stream.connector=quarkus-http
 mp.messaging.incoming.kogito_incoming_stream.path=/
 mp.messaging.incoming.kogito_incoming_stream.method=POST
 
-
+# This property is used when sending the notification while waiting for Q&A
 move2kube_url=${MOVE2KUBE_URL:http://move2kube-svc.default.svc.cluster.local:8080}
+# This property is used to send requests to the move2kube instance
 quarkus.rest-client.move2kube_yaml.url=${MOVE2KUBE_URL:http://move2kube-svc.default.svc.cluster.local:8080}
-broker_url=${BROKER_URL:http://broker-ingress.knative-eventing.svc.cluster.local/m2k/default}
+# This property is used to send requests to the backstage notification plugin
```

Comment:
what about the BACKSTAGE_NOTIFICATIONS_URL below?

Reply:
what about it?

Reply:
> how should it be provided 
Either provide the value through env var in the pod spec template if the props is not overridden or in the props CM if the conf is overridden  
> shouldn't the default value be changed to fit the deployment target resource?
I would not know what to put TBH, we are not in the charts repo so there is no link between backstage URL and M2K. The idea with providing the default value is to show the expected URL (ie: ends with `api/notifications/`).
But, yes, I could put `orchestrator-backstage.orchestrator/api/notifications/` to serve the same purpose and be clearer